app/views/ChargeSummary.scala.html

```diff
@@ -173,6 +168,19 @@
     }
 }
 
+@reviewAndReconcilePoaCreditExplanation(oneOrTwo: String) = {
+    @p(id = Some("rar-credit-explanation")) {
+        @messages(s"chargeSummary.poa${oneOrTwo}Credit.p1")
+        @link(id = Some("poa-link"), link = {
+            oneOrTwo match {
+                case "1" => viewModel.poaOneChargeUrl
```

Comment:
"1" and "2" would be a magic numbers, I guess it will be hard to support this code in the future, maybe we can pass subTransactionType on this occasion if you like to optimise this part of the code

Reply:
I think at the moment subtransactiontype can only be Nics2, Accepted or Rejected. I'm not familiar with the logic behind this, would adding Poa1 and Poa2 interfere with any of that?

Reply:
I think changing "1" and "2" to something more readable would help

Reply:
Have changed these to pass the PaymentOnAccountOne/Two ChargeTypes